io.sourceforge.clonekeenplus.yml

```diff
@@ -0,0 +1,33 @@
+app-id: io.sourceforge.clonekeenplus
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: CGeniusExe
+rename-desktop-file: cgenius.desktop
+rename-icon: cg
+
+modules:
+  - name: cgenius
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://gitlab.com/Dringgstein/Commander-Genius.git
```

Comment:
This has to be a tagged release/tarball or a specific commit at least.

Reply:
You are right. Sorry for overseeing that. Just pushed another commit.